### PR TITLE
Ocp client methods

### LINKED ISSUE #30: `Ocp` methods

### PR DESCRIPTION
**Pull request by phemmylintry**
closes #30

### CODE REVIEW

**Comment by phemmylintry** [comment]
@parisk @akalipetis Please review

File: billit/client.py [hunk 1]
```diff
@@ -449,3 +450,89 @@ def update(
 
     def delete(self, contact_id: int):
         return self.client._handle_request("DELETE", f"/contacts/{contact_id}")
+
+
+class Ocp(SubClient):
```
Comment (parisk) [inline billit/client.py:455]: Let's rename this to OCP, since it's an abbreviation.

File: billit/client.py [hunk 2]
```diff
@@ -26,6 +26,7 @@ def __init__(self, api_key, environment=PRODUCTION_ENVIRONMENT):
         self.invoices = Invoices(self)
         self.customers = Customers(self)
         self.contacts = Contacts(self)
+        self.ocp = Ocp(self)
```
Comment (parisk) [inline billit/client.py:29]: Update according to https://github.com/withlogicco/billit-python-sdk/pull/38/files#diff-f5cff635bc137e72aa003a9abb9891dfabc0c0a1fcaeaf400b38f8c6ce36215eR455

**Review by parisk — CHANGES_REQUESTED** [review]
Great job @phemmylintry! Just one update and we are good to go 😄.

**Review by parisk — APPROVED** [review]
LGTM 🚀

**Pull request merged**